lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -514,11 +494,12 @@ increaseZeroAdaOutputs era pp = modifyLedgerBody $
 
 -- | Internal helper to 'balanceTransaction'
 balanceTransactionWithSelectionStrategyAndNoZeroAdaAdjustment
-    :: forall era m changeState.
-        ( MonadRandom m
-        , IsRecentEra era
-        )
+    :: forall era m changeState
+     . ( MonadRandom m
+       , IsRecentEra era
+       )
```

Comment:
Same comment as above. ^^^
(Propose to remove this change.)

Reply:
(I think this one is still unresolved.)

Reply:
Done

Reply:
Still unresolved